build.sbt

```diff
@@ -7,27 +7,27 @@ val V = new {
   val `cats-effect`      = "3.5.1"
   val circe              = "0.14.5"
   val ciris              = "3.2.0"
-  val confluent          = "7.2.2"
+  val confluent          = "7.4.1"
```

Comment:
stuck here as without a newer version embedded-kafka-schema-registry is binary incompatible with 3.5.x, see https://github.com/embeddedkafka/embedded-kafka-schema-registry/actions/runs/5557892746/job/15064933555?pr=400

Reply:
See also https://github.com/embeddedkafka/embedded-kafka-schema-registry/pull/418

Reply:
See also https://github.com/tchiotludo/akhq/pull/1554